sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
+NewTabTweaker.Notifications = {
```

Comment:
To make this into a requirejs module, we need to call `define(exports)`, with `exports` being the object we wish to be available to users of this module. In our case this is just `{ notify: ... }`. See a minimal example [here](https://github.com/rharel/webext-new-tab-tweaker/blob/master/sources/url_overrides/newtab/js/rng.js).

For this module to be accessible in `image_context_menu.js`, we need to add it as a dependency `"background_scripts/notifications"` [here](https://github.com/rharel/webext-new-tab-tweaker/blob/e55f7f48937ee2f36e2616edaffd73b29c2ea810/sources/background_scripts/image_context_menu.js#L97), similarly to how we import the configuration module.